src/content/docs/kubernetes-pixie/kubernetes-integration/installation/k8s-agent-operator.mdx

```diff
@@ -100,13 +100,19 @@ Each namespace requires a two-step process to enable automatic instrumentation o
 
    Make sure to replace `<YOUR_NEW_RELIC_INGEST_LICENSE_KEY>` with your valid New Relic license key.
 
-2. Apply the following CRD to each namespace you want to auto instrument. Run this command:
+2. Apply the following Custom Resource Definition (CRD) to each namespace you want to auto instrument. Run this command:
```

Comment:
K8s seems to stylize this as `CustomResourceDefinition`. Do we want to follow this?

Reply:
https://kubernetes.io/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions/